### PR TITLE
doc: Make all the tables in SML page manual

### PR DESCRIPTION
**Pull request by umapraseeda**
NCSDK-27739
We have had issues with incorrect SML table values due to the update being through nightly build,
especially in release branches. So  until the script is optimized, the only way to fix this issue is to make the tables manual (grid tables). This PR converts the present values into manual table values.

### CODE REVIEW

**Comment by koffes** [comment]
Not my area, so I will tread carefully. As I have understood, removing this feature will create more manual work for teams and may lead to the status in doc not reflecting the actual status. I do request that is is carefully considered if this is the preferred option.

**Review by divipillai — APPROVED** [review]

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 1]
```diff
@@ -170,27 +170,150 @@ Protocol support
...
+     - --
+     - --
+     - --
+     - --
+     - --
+     - Experimental
+     - Experimental
+     - --
+     - Experimental
+   * - **LTE**
```
Comment (gmarull) [inline doc/nrf/releases_and_maturity/software_maturity.rst:233]: list tables may be difficult to track (finding which line corresponds to which column)

Comment (umapraseeda) [reply doc/nrf/releases_and_maturity/software_maturity.rst:233]: But if it is grid table, doesn't it make it more difficult to make a change? You need to make space alignments for the entire table for a change in one of the values/entries.

**Review by shanthanordic — APPROVED** [review]

**Review by richabp — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 2]
```diff
@@ -222,64 +531,602 @@ The following table indicates the software maturity levels of the support for ea
...
+      * - **Thread FTD + Bluetooth LE multiprotocol**
+        - --
+        - --
+        - --
+        - --
+        - --
+        - Supported
+        - Supported
+        - Experimental
+        - Experimental
```
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/software_maturity.rst:673]: @wiba-nordic please check.

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 3]
```diff
@@ -222,64 +531,602 @@ The following table indicates the software maturity levels of the support for ea
...
+      * - **Matter - OTA DFU over Bluetooth LE**
+        - --
+        - --
+        - --
+        - --
+        - --
+        - Experimental
+        - Experimental
+        - Experimental
+        - Experimental
```
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/software_maturity.rst:757]: @wiba-nordic please check.

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 4]
```diff
@@ -222,64 +531,602 @@ The following table indicates the software maturity levels of the support for ea
...
+        - --
+        - --
+      * - **Thread FTD + Bluetooth LE multiprotocol**
+        - --
+        - --
+        - --
+        - --
+        - --
+        - Experimental
+        - Experimental
```
[the remark this answers is not in the document]
Comment (umapraseeda) [reply doc/nrf/releases_and_maturity/software_maturity.rst]: Updated after checking with Wille.

File: doc/nrf/releases_and_maturity/software_maturity.rst (hunk 3 above)
Comment (wiba-nordic) [reply doc/nrf/releases_and_maturity/software_maturity.rst:757]:
```suggestion
        - Supported
        - Supported
        - Experimental
        - Experimental
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]
Thread was correct.

File: doc/nrf/releases_and_maturity/software_maturity.rst (hunk 3 above)
Comment (umapraseeda) [reply doc/nrf/releases_and_maturity/software_maturity.rst]: fixed

**Review by wiba-nordic — APPROVED** [review]

**Review by richabp — APPROVED** [review]

**Comment by umapraseeda** [comment]
@b-gent and @gmarull, could you check and approve?

**Review by gmarull — APPROVED** [review]

**Review by shanthanordic — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Comment by umapraseeda** [comment]
Looks like the Sidewalk value changed again in main due to the script, @b-gent/ @annwoj could you confirm if all are experimental for this table?

**Comment by annwoj** [comment]
> Looks like the Sidewalk value changed again in main due to the script, @b-gent/ @annwoj could you confirm if all are experimental for this table?

For Sidewalk it was experimental and for the 2.8 the plan is to have everything supported apart from FUOTA, but that's something we will know for sure closer to the release, and that's when we will also apply the changes. so let's keep it as experimental

**Pull request merged**